src/Clean.Architecture.Web/Api/ProjectsController.cs

```diff
@@ -66,10 +67,10 @@ public async Task<IActionResult> Post([FromBody] CreateProjectDTO request)
         var createdProject = await _repository.AddAsync(newProject);
 
         var result = new ProjectDTO
-        {
-            Id = createdProject.Id,
-            Name = createdProject.Name
-        };
+        (
+            id: createdProject.Id,
+            name: createdProject.Name
```

Comment:
Why this change?

Reply:
To satisfy the nullable warnings. If the fields aren't declared as nullable there is a warning if they aren't being set in the constructor. The alternative would have been to just set the fields nullable, however I didn't really see a reason to do that.

Reply:
I'm assuming to ensure non-null by going through the constructor.